apps/web/src/components/createFormTemplate/createFormTemplateEditor/DraggableText.tsx

```diff
@@ -0,0 +1,57 @@
+import { Rnd, RndResizeCallback } from 'react-rnd';
+import { DraggableEventHandler } from 'react-draggable';
+import { TextFieldPosition } from './FormEditor';
+import { FaTimes } from 'react-icons/fa';
+
+export default function DraggableText({
+  onStop,
+  onResizeStop,
+  color,
+  onRemove,
+  currentPosition,
+}: {
+  onStop: DraggableEventHandler;
+  onResizeStop: RndResizeCallback;
+  initialText: string | null;
+  color: string;
+  onRemove: () => void;
+  currentPosition: TextFieldPosition;
+}) {
+  return (
+    <Rnd
+      bounds="parent"
+      position={{ x: currentPosition.x, y: currentPosition.y }}
+      size={{ height: currentPosition.height, width: currentPosition.width }}
```

Comment:
I'm going to set the minHeight to 20 since default is currently 30 when you make a new textbox and 30px fits the text nicely if you test it out. also setting minWidth to 60